### PR TITLE
scss support

### PR DESCRIPTION
**Pull request by torbenbr**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no 
| Tests pass?   | yes    
| Fixed issues  | chameleon-system/chameleon-system#454
| License       | MIT

### CODE REVIEW

File: src/ViewRendererBundle/Compiler/Stylesheet/Adapter/AbstractCompilerAdapter.php [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    {
+        return '/(.*)chameleon_([0-9]{1,16}).css/';
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @todo refactoring
+     */
+    public function writeCssFileForPortal($generatedCss, TdbCmsPortal $portal)
```
Comment (pixeljunkie) [inline src/ViewRendererBundle/Compiler/Stylesheet/Adapter/AbstractCompilerAdapter.php]: writeCssFileForPortal(string $generatedCss, TdbCmsPortal $portal)

**Comment by pixeljunkie** [comment]
The changes should use PHP 7 typehints for return values and parameters (string)

**Comment by pixeljunkie** [comment]
Add an update, that renames the themes LESS index field to SCSS/LESS (the sql name should not change to prevent BC breaks).

File: src/ViewRendererBundle/Compiler/Stylesheet/Adapter/AbstractCompilerAdapter.php (hunk 1 above)
Comment (torbenbr) [reply src/ViewRendererBundle/Compiler/Stylesheet/Adapter/AbstractCompilerAdapter.php]: done

**Pull request closed without merging**